### PR TITLE
JDBC Tasks of Marian Zlatev Tasks 5

### PR DESCRIPTION
**Pull request by NewestUser**
Created a history trigger that stores information upon new insertion. Implemented a class that is used to test triggers behavior.

### CODE REVIEW

**Comment by mgenov** [comment]
I cannot see schema in pull request.

File: schema/examples.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+CREATE TABLE articles
+(
+    id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    title VARCHAR(30) NOT NULL
+);
+CREATE TABLE articles_history
```
Comment (mgenov) [inline schema/examples.sql]: I don't see the code of the trigger.

File: src/com/clouway/jdbc/ArticleRepository.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    try {
+      Statement stmt = connection.createStatement();
+      stmt.execute("INSERT INTO articles (title) values('" + title + "')");
+      stmt.close();
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
+  }
+
+  public List<String> lookupArticles() {
```
Comment (mgenov) [inline src/com/clouway/jdbc/ArticleRepository.java]: findArticles is more common naming pattern. Lookup is is used when hashes as used, i.e you are using lookup table to check where given key contains associated value.

Comment (NewestUser) [inline src/com/clouway/jdbc/ArticleRepository.java]: Renamed lookupArticles to findArticles in order to follow common naming conventions.

File: schema/examples.sql (hunk 1 above)
Comment (NewestUser) [inline schema/examples.sql]: Trigger syntax is now added in schema.

**Pull request merged**